models/mixpanel__daily_events.sql

```diff
@@ -127,18 +118,14 @@ final as (
         number_of_users - number_of_new_users - number_of_repeat_users as number_of_return_users,
         trailing_users_28d,
         trailing_users_7d,
-        event_type || '-' || date_day as unique_key
+        {{ dbt_utils.generate_surrogate_key(['event_type', 'date_day']) }} as unique_key,
```

Comment:
That's a good point. I wanted to be consistent with some of the other models, but it makes sense to keep readablity. I will put it back!
```suggestion
        event_type || '-' || date_day as unique_key,
```

Reply:
Sounds great, thanks!